packages/graph/hash_graph/lib/graph/src/ontology/types/mod.rs

```diff
@@ -26,8 +25,3 @@ pub use crate::ontology::types::{
 pub mod error;
 
 mod serde_shared;
-
-pub trait OntologyType {
-    /// Returns the unique versioned URI used to identify this instance of a type.
-    fn uri(&self) -> &VersionedUri;
-}
```

Comment:
Moved to `OntologyDatabaseType`, this trait was unused in a generic context outside of the database. For concrete types, `.id()` is available. `uri()` was confusing with the `uri()` function on `OntologyQuery`